src/utils.ts

```diff
@@ -3,6 +3,6 @@ export const normalize = (str: string) => {
         .toLowerCase()
         .normalize('NFD')
         .replace(/[\u0300-\u036f]/g, '')
-        .replace(/[^a-zA-Z0-9\-\:\;\(\)\.\,\/\s]/g, '')
+        .replace(/[^a-zA-Z0-9\s]/g, ' ') // replacing any special char with a space instead of an empty string should avoid game names like "NieR:Automata™" to end up as "NieRAutomata" which will fail in HLTB search
```

Comment:
I did not see any case where handling some special characters differently was necessary. From my tests, replacing any special chars by a space instead of an empty string fixes many issues like the ones seen with "Risen 3 - Titan Lords", "NieR:Automata" or "Legacy of Kain Soul Reaver 1&2 Remastered". Test file was not yet updated to reflect this change

Reply:
This is a better solution than constantly adding special characters into the exclusion list 😃.

One addition that I may suggest here - is to add replacing of multiple consequent whitespace characters by just one whitespace character. That should keep string even more "normalized" and possibly keep us from extra problems caused by any upcoming HLTB API regression. Plugin splits game name by space here: 
https://github.com/morwy/hltb-for-deck/blob/e6627ac3892029f81283dc4a585ee1bbeb95ec89/src/hooks/HltbApi.ts#L135

Who knows how HLTB backend will react to an empty search term in case their piece of code getting badly refactored?